tests/test_keras_integration.py

```diff
@@ -300,7 +300,7 @@ def test_abs_path_from_pretrained(self):
             f"{WORKING_REPO_DIR}/{REPO_NAME}",
             config={"num": 10, "act": "gelu_fast"},
             plot_model=True,
-            task_name=None,
+            tags=None,
```

Comment:
You should likely also add a test that asserts that a warning is raised when people use `task_name`